### DISCUSSION TITLE
Pr6048e Inverter added

### DISCUSSION THREAD

**Discussion by tecnicounosuper** [slipx06/sunsynk-power-flow-card#583] [General]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I would like to add plant that we use. There are already many of us. To use your own script
The flowchart. 
Composed of regulator panels they go on equipotential where we have multiple batteries up to 32 that we manage either via BMS or shantt. I'm willing to give you all the schemes

### Expected behaviour

Inverters can be added pr6048e Or called rpseries. Below I will give you some photos of the parameters. I am available to debag and create user on my has. 

### Possible Solutions

Thank you

### Mode

full

### Context / Reason

If you in time our group would be happy we also have 3 Italian YouTubers. To increase diffusion and verification of operation. 
![Screenshot_20241123_082035_Home Assistant](https://github.com/user-attachments/assets/49d2749d-6641-4663-9e2f-d6babb3d655f)
![20241123_083154](https://github.com/user-attachments/assets/2d608c5c-daf4-476e-8423-b879cce680b2)
![Screenshot_20241123_083307_Home Assistant](https://github.com/user-attachments/assets/f7b403e0-20a1-4eca-8ffe-8711af315327)
![Screenshot_20241123_083316_Home Assistant](https://github.com/user-attachments/assets/c9ea9572-a474-45d8-8188-a586b140eac2)
I am attaching photos of the inverter data. And photos of the data of the schant. From BMS we have the usual readings. 
I remain available thank you

**Comment by slipx06** [comment]
HI. You should already be able to use the card to display you sensor data. Have you tried to map some of your data?
